YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.

I tried to run the tests of addon [volto-columns-block](https://github.com/eea/volto-columns-block) locally with the `yarn test` command, but I get the error:

```bash
➤ YN0000: └ Completed in 6s 686ms
➤ YN0000: ┌ Post-resolution validation
➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
➤ YN0000: └ Completed
➤ YN0000: Failed with errors in 6s 837ms
make: *** [Makefile:46: test] Erro 1
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This addon uses [volto-addon-ci](https://github.com/eea/volto-columns-block/blob/b4c52143641c95695ef266b89bd2b1a4fd9bb5c0/Makefile#L46).

I researched this problem and, from what I understood, it occurs when the node configuration:

`enableImmutableInstalls`

is `true`

and there is a change in the `yarn.lock` file when we run yarn: See:

https://stackoverflow.com/questions/67062308/getting-yn0028-the-lockfile-would-have-been-modified-by-this-install-which-is-e

But the probability of `yarn.lock` being changed is very high. For example, the addon may have a specific dependency. So I propose to force the use of `enableImmutableInstalls=false` by running the command:

`YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn`

--- a/.gitignore
+++ b/.gitignore
@@ -1,1 +1,3 @@
 *~
+.*project
+.settings/

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,7 @@
 
 ### Unreleased
 
+- Force enableImmutableInstalls=false when running the yarn command [#11](https://github.com/plone/volto-addon-ci/issues/11)
 - Copy <addon>/.project.eslintrc to <project>/.eslintrc.js [#9](https://github.com/plone/volto-addon-ci/pull/9)
 - Handle add-on resolutions [`8`](https://github.com/plone/volto-addon-ci/pull/8)
 - increase default cypress start project timeout [`#5`](https://github.com/plone/volto-addon-ci/pull/5)

--- a/docker-entrypoint.sh
+++ b/docker-entrypoint.sh
@@ -87,7 +87,8 @@ if [ "$resolutions" != "null" ]; then
   mv package.json.res package.json
 fi
 
-yarn
+# Allows the yarn.lock file to be changed.
+YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn
 
 if [[ "$1" == "test"* ]]; then
   exec bash -c "set -o pipefail; RAZZLE_JEST_CONFIG=$RAZZLE_JEST_CONFIG CI=true yarn test src/addons/$GIT_NAME/src --watchAll=false --reporters=default --reporters=jest-junit --collectCoverage --coverageReporters lcov cobertura text 2>&1 | tee -a unit_tests_log.txt"

